usage/src/main/java/com/cloud/usage/UsageManagerImpl.java

```diff
@@ -2162,6 +2177,13 @@ private void handleVpcEvent(UsageEventVO event) {
         if (EventTypes.EVENT_VPC_DELETE.equals(event.getType())) {
             usageVpcDao.remove(event.getResourceId(), event.getCreateDate());
         } else if (EventTypes.EVENT_VPC_CREATE.equals(event.getType())) {
+            List<UsageVpcVO> entries = usageVpcDao.listAll(event.getResourceId());
+            if (!entries.isEmpty()) {
+                s_logger.warn(String.format("Active helper entries already exist for VPC [%s]; therefore, we will not create a new one.",
+                        event.getResourceId()));
+                return;
+            }
+            s_logger.debug(String.format("Creating a helper entry for VPC [%s].", event.getResourceId()));
             UsageVpcVO usageVPCVO = new UsageVpcVO(event.getResourceId(), event.getZoneId(), event.getAccountId(), domainId, Vpc.State.Enabled.name(), event.getCreateDate(), null);
```

Comment:
could be a new method (again, return needs to be handled with a checked exception)